guides/client/dom-patching.md

```diff
@@ -11,12 +11,18 @@ own DOM operations. The following `phx-update` values are supported:
   * `append` - append the new DOM contents instead of replacing
   * `prepend` - prepend the new DOM contents instead of replacing
 
-When using `phx-update`, a unique DOM ID must always be set in the
-container. If using "append" or "prepend", a DOM ID must also be set
-for each child. When appending or prepending elements containing an
-ID already present in the container, LiveView will replace the existing
-element with the new content instead appending or prepending a new
-element.
+> ### Unique DOM IDs {: .info}
+>
+> When using `phx-update`, a unique DOM ID must always be set for the
+> container. If using `append` or `prepend`, a DOM ID must also be set
+> for each child. When appending or prepending elements containing an
+> ID already present in the container, LiveView will replace the existing
+> element with the new content instead appending or prepending a new
+> element.
+>
+> Note that if rendering moves a container from *outside* the container
+> with `phx-update` to *inside* the container with `phx-update`, you must 
+> also give the moved container a new DOM ID.
```

Comment:
Oh, I see. This part may not be intentional and perhaps we should do a check explicitly on the container before. /cc @chrismccord

Reply:
👍 I do think it might still be good to put the dom id thing in a call-out since I see this issue float by all the time on elixirforums e.g.